drivers/builder/windows/vs2019/builder/builder.vcxproj

```diff
@@ -47,7 +47,7 @@
       <IncludePath>..\..\..\..\..\deps\hypervisor\bfsdk\include;..\..\..\..\..\deps\hypervisor\bfelf_loader\include;..\..\..\..\..\include;..\..\..\..\..\deps\xen\xen\include\public;..\..\..\;..\..\;$(IncludePath)</IncludePath>
   </PropertyGroup>
   <PropertyGroup>
-    <SignMode>Off</SignMode>
+    <SignMode>TestSign</SignMode>
```

Comment:
This property needs to be Off, otherwise the test cert shows up in the driver's .sys properties file

Reply:
Oh I see that it is toggled in microv.yml